cpp/tests/parser_test.cc

```diff
@@ -9,7 +9,7 @@
 
 TEST_CASE("Yaml parser and setting variables test")
 {
-  std::string file_path = "../data/config/config.yaml";
+  std::string file_path = "../../../data/config/config.yaml";
```

Comment:
use the function `purify::notinstalled::data_filename("config/config.yaml");` You will need to include `#include "purify/directories.h"`

Reply:
what does it do ? Does it return a path ?

Reply:
done